From 988cd7a6bcbb738d6ad334a94fda07c965da862e Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Wed, 5 Dec 2018 16:31:13 +0800
Subject: [PATCH] sdk_dpaa: remove useless 1588 timer enablement

1588 timer had been enabled to run at ptp_qoriq driver
probing stage. So removed the useless enablement.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c | 4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
@@ -340,8 +340,6 @@ static void dpa_ts_tx_enable(struct net_
 	struct dpa_priv_s *priv = netdev_priv(dev);
 	struct mac_device *mac_dev = priv->mac_dev;
 
-	if (mac_dev->fm_rtc_enable)
-		mac_dev->fm_rtc_enable(get_fm_handle(dev));
 	if (mac_dev->ptp_enable)
 		mac_dev->ptp_enable(mac_dev->get_mac_handle(mac_dev));
 
@@ -373,8 +371,6 @@ static void dpa_ts_rx_enable(struct net_
 	struct dpa_priv_s *priv = netdev_priv(dev);
 	struct mac_device *mac_dev = priv->mac_dev;
 
-	if (mac_dev->fm_rtc_enable)
-		mac_dev->fm_rtc_enable(get_fm_handle(dev));
 	if (mac_dev->ptp_enable)
 		mac_dev->ptp_enable(mac_dev->get_mac_handle(mac_dev));
 
